code/modules/admin/verbs/randomverbs.dm

```diff
@@ -802,8 +798,6 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		message_admins("<span class='notice'>[key_name_admin(usr)] used gibself.</span>", 1)
 		if(!istype(mob, /mob/dead/observer))
 			mob.gib()
-		else
-			gibs(mob.loc, mob.viruses)
```

Comment:
Is there a reason in particular this PR removed admin observer gibself?

Reply:
probably something about mob.viruses being deprecated

Reply:
Also if admins want to spawn gibs they can easily spawn a gibspawner, so I wasn't sure of the point of this proc, aside from clogging up the admin logs/chat window.